internal/impl/babysitter.go

```diff
@@ -153,6 +156,7 @@ func (b *babysitter) ActivateComponent(_ context.Context, request *protos.Activa
 		if err := b.watchPods(b.ctx, request.Component); err != nil {
 			// TODO(mwhittaker): Log this error.
 			fmt.Fprintf(os.Stderr, "watchPods(%q): %v", request.Component, err)
+			b.logger.Error("error watching pods", "err", err, "component", request.Component)
```

Comment:
I still think it deserves a panic, or a retry... 
will be really hard to identify an issue here just by monitoring the services.... errors happens all the time, but this is not a recoverable error.